float64 kiva_left_target_speed
float64 kiva_right_target_speed
float64 fOdoX
float64 fOdoY
float64 fOdoPhi
float64 fVx
float64 fVy
float64 fW
